back/app/models/app_configuration.rb

```diff
@@ -24,13 +24,7 @@ class AppConfiguration < ApplicationRecord
   accepts_nested_attributes_for :text_images
 
   validates :settings, presence: true, json: {
-    schema: -> { AppConfiguration::Settings.json_schema_str },
-    message: lambda { |errors|
-               errors.map do |e|
-                 { fragment: e[:fragment], error: e[:failed_attribute], human_message: e[:message] }
-               end
-             },
-    options: { errors_as_objects: true }
+    schema: -> { AppConfiguration::Settings.json_schema_str }
```

Comment:
Maybe we can use
```suggestion
    schema: -> { AppConfiguration::Settings.json_schema }
```
then? If so, I wonder whether there still a need for `json_schema_str`.

Reply:
Thanks, changing it indeed. Seems like json_schema_str is coming from a mixin that's also used somewhere else, so won't remove it.